homeassistant/components/homeassistant_sky_connect/strings.json

```diff
@@ -68,12 +112,92 @@
+    },
+    "progress": {
+      "install_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::progress::install_addon%]",
+      "start_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::progress::start_addon%]",
+      "start_otbr_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::progress::start_addon%]",
+      "install_zigbee_flasher_addon": "[%key:component::homeassistant_sky_connect::config::progress::install_zigbee_flasher_addon%]",
+      "run_zigbee_flasher_addon": "[%key:component::homeassistant_sky_connect::config::progress::run_zigbee_flasher_addon%]",
+      "uninstall_zigbee_flasher_addon": "[%key:component::homeassistant_sky_connect::config::progress::uninstall_zigbee_flasher_addon%]"
+    }
+  },
+  "config": {
+    "flow_title": "{model}",
+    "step": {
+      "confirm": {
+        "title": "Set up the {model}",
+        "description": "The {model} can be used as either a Thread border router or a Zigbee coordinator. In the next step, you will choose which firmware will be configured."
+      },
+      "pick_firmware": {
+        "title": "Pick your firmware",
+        "description": "The {model} can be used as a Thread border router or a Zigbee coordinator.",
+        "menu_options": {
+          "pick_firmware_thread": "Use as a Thread border router",
+          "pick_firmware_zigbee": "Use as a Zigbee coordinator"
+        }
+      },
+      "install_zigbee_flasher_addon": {
+        "title": "Installing flasher",
+        "description": "Installing the Silicon Labs Flasher add-on."
+      },
+      "run_zigbee_flasher_addon": {
+        "title": "Installing Zigbee firmware",
+        "description": "Installing Zigbee firmware. This will take about a minute."
+      },
+      "uninstall_zigbee_flasher_addon": {
+        "title": "Removing flasher",
+        "description": "Removing the Silicon Labs Flasher add-on."
+      },
+      "zigbee_flasher_failed": {
+        "title": "Zigbee installation failed",
+        "description": "The Zigbee firmware installation process was unsuccessful. Ensure no other software is trying to communicate with the {model} and try again."
+      },
+      "confirm_zigbee": {
+        "title": "Zigbee setup complete",
+        "description": "Your {model} is now a Zigbee coordinator and will be shown as discovered by the Zigbee Home Automation integration once you exit."
+      },
+      "install_otbr_addon": {
+        "title": "Installing OpenThread Border Router add-on",
+        "description": "The OpenThread Border Router (OTBR) add-on is being installed."
+      },
+      "start_otbr_addon": {
+        "title": "Starting OpenThread Border Router add-on",
+        "description": "The OpenThread Border Router (OTBR) add-on is now starting."
```

Comment:
These titles and descriptions say the exact same thing. I think that we can fold "Starting" into "Install". The description can maybe pitch what the OTBR actually does?